keras_cv/bounding_box/iou.py

```diff
@@ -17,7 +17,13 @@
 from keras_cv import bounding_box
 
 
-def compute_iou(boxes1, boxes2, bounding_box_format):
+def compute_iou(
```

Comment:
thanks for upstreaming this.

QQ:
- should we default `use_masking` to True.

Reply:
We use this as a "trick" which is not normally something done in iou compute, so leaving as False (consistent with original behavior)